haskell-language-server.cabal

```diff
@@ -108,6 +108,55 @@ library
 
   default-language: Haskell2010
 
+flag ghc-lib
+  default: False
+  manual: True
+  description: Force dependency on ghc-lib-parser even if GHC API in the ghc package is supported
+
+library hls-ghc-lib
```

Comment:
Does stack cradle work with private library now? I thought it hasn't been fixed.

Reply:
No, it won't, thanks for pointing it out, i forgot to warn about that in pr description. Unfortunately afaik we cant convert it in a common stanza cause the main reason is to be able to have its own flags and dependencies.
There are plans to allow plugins live in its own package, i think this one could be one of them.

Reply:
Thank you for the clarification. Could you direct me to the description of the plan? I may take and/or tackle some parts of the plan.

Reply:
@Ailrun sure: #164